### PR TITLE
Network binding plugins: Add compute resource overhead

### PR DESCRIPTION
**Pull request by orelmisan**
**What this PR does / why we need it**:
Following PR kubevirt/kubevirt#12235, it is now possible to specify memory overhead for the virt-launcher's compute container.

Add an explanation about the compute resource overhead in general, with a note that only memory overhead request is currently supported.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
None
```

### CODE REVIEW

**Review by EdDev — APPROVED** [review]
Thank you!

**Review by ormergi — COMMENTED** [review]
Thanks for the PR 

I had one comment about the sentence that the scribe the end result of specifying the new feild, and some suggestions for rephrasing (these are for you to consider)

File: docs/network/network_binding_plugins.md [hunk 1]
```diff
@@ -208,6 +208,43 @@ kubectl patch kubevirts -n kubevirt kubevirt --type=json -p='[{"op": "add", "pat
...
+                      "requests": {
+                        "memory": "500Mi",
+                      }
+                    }
+                }
+            }
+        }}]'
+```
+
+Every compute container in a virt-launcher pod derived from a VM using the passt network binding plugin, will have an additional `500Mi` memory overhead.
```
[the remark this answers is not in the document]
Comment (orelmisan) [reply docs/network/network_binding_plugins.md:246]: Thank you for the suggestion, I think advanced users of KubeVirt are aware of the fact that there is only a single compute container per virt-launcher pod.

**Review by phoracek — COMMENTED** [review]
/approve

**Pull request merged**